scss/components/_day-sale.scss

```diff
@@ -0,0 +1,126 @@
+.day-sale {
+  padding-bottom: 207px;
+  height: 1000px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-end;
+    padding-right: 35px;
+  }
+
+  //Info (left side)
```

Comment:
what does this comment mean? Please don't leave meaningless comments in your code